docker-files/database/init-files/test_mysql_init.sql

```diff
@@ -199,6 +199,40 @@ INSERT INTO "customers" ("customer_id","name","email","prefecture","city","addre
 ('ac1d4562-801c-ea25-c09d-fbd0838467d9','波留四郎','sample4@gmail.com','大阪','大阪市','1-23-7','8931104','7daf403c7589f4927632ed3b6af762a992f09b78','2024-11-5 12:00','2024-11-5 12:00'),
 ('506658a7-9b1b-3b35-5d81-0e616684a744','波留五郎','sample5gmail.com','福岡','博多市','1-23-8','8931105','4f1cef8d900db702b7759ef360430fd6151362a1','2024-11-5 12:00','2024-11-5 12:00');
 
+CREATE TABLE "images"{
+	"image_id" UUID NOT NULL UNIQUE DEFAULT gen_random_uuid(),
+	"url" VARCHAR(255) NOT NULL,
+}
```

Comment:
```suggestion
CREATE TABLE "images"(
	"image_id" UUID NOT NULL UNIQUE DEFAULT gen_random_uuid(),
	"url" VARCHAR(255) NOT NULL
)
```
書式ミス。丸括弧で囲むのと、最後の行にカンマは不要。